scribejava-apis/src/main/java/com/github/scribejava/apis/FacebookApi.java

```diff
@@ -42,4 +52,32 @@ public String getAuthorizationUrl(final OAuthConfig config) {
         }
         return sb.toString();
     }
+
+    @Override
+    public OAuth20Service createService(OAuthConfig config) {
+        return new FacebookOAuth20ServiceImpl(this, config);
+    }
+    
+    private static class FacebookOAuth20ServiceImpl extends OAuth20Service {
```

Comment:
Yep. It's better to have separate file with service extending OAuth20Service. As it is with all other APIs and Services